src/index.js

```diff
@@ -0,0 +1,40 @@
+document.getElementById("up-button").onclick = function(){
+    var tempInt = parseInt(document.getElementById("temperature").innerHTML);
+    tempInt++;
+    document.getElementById("temperature").innerHTML = tempInt;
+    tempColor(tempInt);
+}
+
+document.getElementById("down-button").onclick = function(){
+    var tempInt = parseInt(document.getElementById("temperature").innerHTML);
+    tempInt--;
+    document.getElementById("temperature").innerHTML = tempInt;
+    tempColor(tempInt);
+}
+
+function tempColor(temp) {
+        let color = undefined;
+        let floor = undefined;
+        if (temp >= 80) {
+          color = 'red';
+          floor = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+        } else if (temp >= 70) {
+          color = 'orange';
+          floor = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+        } else if (temp >= 60) {
+          color = 'yellow';
+          floor = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+        } else if (temp >= 50) {
+          color = 'green';
+          floor = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+        } else {
+          color = 'teal';
+          floor = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+        }
```

Comment:
Consider the above logic and how similar all the branches are. We look for whether the temperature is within a range, and pick a color and landscape accordingly. What if we had a list of records that we could iterate through to find the values. We could set up something like
```js
  const tempRecords = [
    { lowerBound: 80, color: "red", landscape: "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂"},
    { lowerBound: 70, color: "other color", landscape: "other landscape"},
    ...
  ];
```

For the `else` case, we might set the `lowerBound` to a very low value, or to `null` and write a special case to check it. But for everything else, we could simply iterate through the list and find the first record that has a lower bound _lower_  than our temperature, then use it as the source of picking the color and landscape!